Chess/Chess/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Chess
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("White Rook Start");
+            var poswhite = Console.ReadLine().ToLower();
```

Comment:
Для имен переменных лучше использовать принцип _camelCase_, так легче читать код
```suggestion
            var posWhite = Console.ReadLine().ToLower();
```